sentinel-core/src/main/java/com/alibaba/csp/sentinel/util/SpiLoader.java

```diff
@@ -219,16 +233,16 @@
                 int order = SpiOrderResolver.resolveOrder(spi);
                 // Since SPI is lazy initialized in ServiceLoader, we use online sort algorithm here.
                 SpiOrderResolver.insertSorted(orderWrappers, spi, order);
-                RecordLog.info("[SpiLoader] Found {0} SPI: {1} with order " + order, clazz.getSimpleName(),
-                        spi.getClass().getCanonicalName());
+                RecordLog.debug("[SpiLoader] Found {} SPI: {} with order {}", clazz.getSimpleName(),
+                        spi.getClass().getCanonicalName(), order);
             }
             List<T> list = new ArrayList<>();
             for (int i = 0; i < orderWrappers.size(); i++) {
                 list.add(i, orderWrappers.get(i).spi);
             }
             return list;
         } catch (Throwable t) {
-            RecordLog.warn("[SpiLoader] ERROR: loadDifferentInstanceListSorted failed", t);
+            RecordLog.warn("[SpiLoader] ERROR: loadPrototypeInstanceListSorted failed", t);
```

Comment:
`warn` and `ERROR` are a little confused(Although it's also not included in this PR actually).
And after several iterations enhancing logging of Sentinel is it still necessary to use `printStackTrace()` directly? We just make a decision and go.

Reply:
Yes, in Exception catch block, use `ERROR` is better, it has been fixed, please check.

> And after several iterations enhancing logging of Sentinel is it still necessary to use printStackTrace() directly? We just make a decision and go.

Use `printStackTrace()` directly doesn't look good indeed, maybe `printStackTrace()` is useful for users to view directly, since `RecordLog` log the message info file by default,
so I haven't remove it, and waiting for your confirm.I also wonder that if we could throw `RuntimeException` in the catch block. 
BTW, the catch block seems to be never happen or extremely small probability.

Reply:
> Yes, in Exception catch block, use `ERROR` is better, it has been fixed, please check.
> 
> > And after several iterations enhancing logging of Sentinel is it still necessary to use printStackTrace() directly? We just make a decision and go.
> 
> Use `printStackTrace()` directly doesn't look good indeed, maybe `printStackTrace()` is useful for users to view directly, since `RecordLog` log the message info file by default, so I haven't removed it, and waiting for your confirm.I also wonder that if we could throw `RuntimeException` in the catch block.
> BTW, the catch block seems to be never happen or extremely small probability.

Yeah actually i have no position here and we could keep it then